iroh/src/node/builder.rs

```diff
@@ -125,6 +125,7 @@ where
     #[debug("callback")]
     gc_done_callback: Option<Box<dyn Fn() + Send>>,
     blob_events: EventSender,
+    transport_config: Option<quinn::TransportConfig>,
```

Comment:
are we reexporting the `TransportConfig` from `iroh-net`? if so maybe use that as import?